src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -794,3 +797,51 @@ def plot_GP_components(lc, model, meta, fitter, isTitle=True):
         fig.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
         if not meta.hide_plots:
             plt.pause(0.2)
+
+
+def plot_fleck_star(lc, model, meta, fitter):
```

Comment:
starry is prohibitively slow with star spots right now, it's technically enabled but practically useless. I can add the plots but not sure they're actually useful.

Reply:
If we're enabling starry spots for v1.0, I think it'd be important to include a plot for them. Hopefully it should require minimal work from you since starry has most of the plotting utilities built-in